test/unit/api/properties.js

```diff
@@ -162,6 +163,55 @@ describe('api/prop', () => {
     });
   });
 
+  describe('slot', () => {
+    let div1, div2, elem;
+
+    beforeEach(() => {
+      elem = new (define('x-test', {
+        props: {
+          test1: prop.slot(),
+          test2: prop.slot({ name: 'test' })
+        }
+      }))();
+      div1 = document.createElement('div');
+      div2 = document.createElement('div');
+      div2.setAttribute('slot', 'test');
+    });
+
+    it('no children', done => {
+      fixture(elem);
+      afterMutations(
+        () => expect(elem.test1).to.deep.equal([]),
+        () => expect(elem.test2).to.deep.equal([]),
+        done
+      );
+    });
+
+    it('existing children', done => {
+      fixture(elem);
+      elem.appendChild(div1);
+      elem.appendChild(div2);
+      afterMutations(
+        () => expect(elem.test1).to.deep.equal([div1]),
+        () => expect(elem.test2).to.deep.equal([div2]),
+        done
+      );
+    });
+
+    it('new children', done => {
+      fixture(elem);
+      afterMutations(
+        () => expect(elem.test1).to.deep.equal([]),
+        () => expect(elem.test2).to.deep.equal([]),
+        () => elem.appendChild(div1),
+        () => elem.appendChild(div2),
+        () => expect(elem.test1).to.deep.equal([div1]),
+        () => expect(elem.test2).to.deep.equal([div2]),
+        done
+      );
+    });
+  });
+
```

Comment:
That passes for me. I copied and updated indentation. This looks right, correct?

```js
describe('nadia breaking my tests again!', () => {
  let setCallOne = sinon.spy();
  let setCallTwo = sinon.spy();
  beforeEach(() => {
    elem = new (define('x-test', {
      props: {
        test1: prop.slot({
          set: setCallOne,
        }),
        test2: prop.slot({ name: 'test',
          set: setCallTwo,
        })
      },
    }))();
    div1 = document.createElement('div');
    div2 = document.createElement('div');
    div2.setAttribute('slot', 'test');
  });
  it('set should work', done => {
    fixture(elem);
    afterMutations(
      () => {
        expect(setCallOne).not.to.be.called;
        expect(setCallTwo).not.to.be.called;
      },
      () => elem.appendChild(div1),
      () => elem.appendChild(div2),
      () => {
        expect(setCallOne).to.be.called;
        expect(setCallTwo).to.be.called;
      },
      done
    );
  });
});
```